npu.user.js

```diff
@@ -506,8 +507,11 @@ var npu = {
 					var select = $('<ul class="termSelect"></ul>');
 					var stored = npu.getUserData(null, null, ["termSelect", npu.getPage()]);
 					var found = false;
+					var admissionSemester = String($("#lblTrainingName").text().match(/:(\d{4}\/\d{2}\/\d)\[.*?\]\)/)[1]);
+					
 					$("option", termSelect).each(function() {
 						if($(this).attr("value") == "-1") { return; }
+						if($(this).text() < admissionSemester) { return; }

```

Comment:
We should prepare for the possibility that no admission semester is matched:

    if(admissionSemester && $(this).text() < admissionSemester) { return; }